Sources/XMTPiOS/Client.swift

```diff
@@ -170,11 +170,6 @@ public final class Client {
 			signingKey: account
 		)
 
-		if let textToSign = v3Client?.textToSign() {
```

Comment:
This seems important. Does it exist somewhere else?

Reply:
It does it's just repeated code.

https://github.com/xmtp/xmtp-ios/blob/main/Sources/XMTPiOS/Client.swift#L143-L152